File: C Program/C++/Assignment03/Rectangle.cpp
#include <iostream>
using namespace std;

class Rectangle
{
    float length;
    float breadth;

public:
    void input()
    {
        cout << "Enter the Parameter to Calculate Area & Perimeter " << endl;
        cin >> length >> breadth;
    }
    void perimeter()
    {
        float p = 2 * (length + breadth);
        cout << "Perimeter of Rectangle " << 2 * (length + breadth) << endl;
    }
    void area()
    {
        float a = length * breadth;
        cout << "Rectangle Area " << length * breadth << endl;
    }
};
int main()
{
    Rectangle r;
    r.input();
    r.perimeter();
    r.area();
}

File: JAVA/Assignment 06/Rectangle.java
/*Using this in Constructors: Create a class Rectangle with attributes length and width.
Implement a parameterized constructor that initializes these attributes.
 Use this to differentiate between class variables and constructor parameters. 
 Include methods to calculate the area and
perimeter.*/
import java.util.*;
public class Rectangle{
	
		 protected double length;
		 protected double width;
		 
		 public Rectangle(double length, double width){
		 	this.length = length;
		 	this.width = width;
		 }
		 public double areaRec(){
		 	return length * width;
		 }
		 public double perimeter(){
		 	return 2 * (length + width);
		 }
		 public void disp(){
		 	System.out.println("Length is : "+length);
		 	System.out.println("Width is  : "+width);
		 }
	}
